### PR TITLE
fix(iOS-gestures) touch delegate does not call base class touch methods

### PR DESCRIPTION
**Pull request by farfromrefug**
## PR Checklist

- [x] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [x] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [x] You have signed the [CLA](http://www.nativescript.org/cla).
- [x] All existing tests are passing: https://github.com/NativeScript/NativeScript/blob/master/DevelopmentWorkflow.md#running-unit-tests.
- [ ] Tests for the changes are included - https://github.com/NativeScript/NativeScript/blob/master/WritingUnitTests.md.

## What is the current behavior?
Attach touch gesture to a button (best example to see the issue). You won't get button animations anymore.

## What is the new behavior?
The touch gesture delegate now calls base methods on the native observed view.

Fixes https://github.com/NativeScript/NativeScript/issues/4449

You can test it here: https://play.nativescript.org/?template=play-ng&id=46hl3N&v=2

### CODE REVIEW

**Comment by vakrilov** [comment]
test

File: tns-core-modules/ui/gestures/gestures.ios.ts [hunk 1]
```diff
@@ -353,18 +353,22 @@ class TouchGestureRecognizer extends UIGestureRecognizer {
 
     touchesBeganWithEvent(touches: NSSet<any>, event: any): void {
         this.executeCallback(TouchAction.down, touches, event);
+        (this.observer.target.nativeView as UIView).touchesBeganWithEvent(touches, event);
```
Comment (vakrilov) [inline tns-core-modules/ui/gestures/gestures.ios.ts]: Can you use `this.view` (type is `UIView`) -> it should be the same, only shorter and without the casting.

**Comment by vakrilov** [comment]
test

**Comment by vakrilov** [comment]
@farfromrefug Looks good - thanks for the PR. Can just check my review.

**Comment by vakrilov** [comment]
test

**Comment by vakrilov** [comment]
test

**Comment by vakrilov** [comment]
Hey @farfromrefug - there is an unstable test in the CI that is causing some of the PR builds to fail. We are currently hunting it down. 
There is nothing you can do - I don't think this is a problem with this PR.

**Comment by SvetoslavTsenov** [comment]
test

**Comment by farfromrefug** [comment]
@vakrilov thanks for explaining. I think I also understand your "test" comments now. It s there to trigger a test build!

**Comment by vakrilov** [comment]
You are correct - test just triggers the internal Jenkins builds.
Although, if you don't know about it, it's easy to take it the wrong way:

**Comment by SvetoslavTsenov** [comment]
test

**Comment by shiv19** [comment]
@farfromrefug Thanks! 😄 
I was staring at that part of `gestures.ios.ts` not being able to understand why it isn't working.
And then I noticed that that file was changed 13 days ago, so the changes aren't in `@latest` version of tns-core-modules.

**Comment by farfromrefug** [comment]
@shiv19 use @next

**Comment by shiv19** [comment]
Yes. But I was trying to use this for a NativeScript snack. Which is on NativeScript Playground.
Because of that, I'll have to wait till this change reaches the `@latest` version, and is upgraded on the playground.

**Pull request merged**